tests/e2e/utils/parser.go

```diff
@@ -39,6 +39,7 @@ type rpcFile struct {
 
 const (
 	DYNAMIC_RPC_FILE = "contract-examples/dynamic_rpc.json"
+	LOCAL_RPC_FILE   = "contract-examples/local_rpc.json"
```

Comment:
To be honest, this makes no sense to me either. I had to add this back in recently to make the tests pass. We run ``cmd := exec.Command("npx", "hardhat", "test", test, "--network", "e2e")``` which should mean that we don't need to write to LOCAL. But CI wasn't passing otherwise.

Reply:
I have some typescript compiler suspicions but we do need both.

Reply:
Ok, let's leave fixing this out of scope for this PR then.